ArcGISRuntimeSDKQt_QMLSamples/Layers/DisplayKMLNetworkLinks/MessageButton.qml

```diff
@@ -0,0 +1,82 @@
+// [WriteFile Name=DisplayKMLNetworkLinks, Category=Layers]
```

Comment:
We have a mix of how we reference Kml. I think this is ok even though the types for Qt are `Kml`.